packages/experimental-utils/src/ts-eslint/RuleTester.ts

```diff
@@ -46,7 +46,7 @@ interface RunTests<
   invalid: InvalidTestCase<TMessageIds, TOptions>[];
 }
 interface RuleTesterConfig {
-  parser: '@typescript-eslint/parser';
+  parser?: '@typescript-eslint/parser' | 'espree'| 'babel-eslint' | 'esprima' | string;
```

Comment:
These are the ones listed here: https://eslint.org/docs/user-guide/configuring#specifying-parser

added `string` as well in case there are others